site_scons/site_tools/roc/config.py

```diff
@@ -290,6 +290,54 @@ def FindConfigGuess(context):
     context.Result('not found')
     return False
 
+def FindPkgConfigPath(context):
+    env = context.env
+
+    context.Message("Searching PKG_CONFIG_PATH...")
+
+    if env.HasArg('PKG_CONFIG_PATH'):
+        env.Result(env['PKG_CONFIG_PATH'])
+        return True
+
+    if env.HasArg('PKG_CONFIG'):
+        pkg_config = env['PKG_CONFIG']
+    elif env.Which('pkg-config'):
+        pkg_config = 'pkg-config'
+    else:
+        return False
+
+    pkg_config_paths = env.CommandOutput('%s --variable pc_path %s' % (pkg_config, pkg_config))
```

Comment:
1. We should quote pkg_config variable because it can contain spaces, like:

```
'"%s" --variable pc_path "%s"' % (pkg_config, pkg_config)
```

or better use `quote()` function like we do it in thirdparty.py.

2. I guess the package name, i.e. the second %s, should be always `pkg-config`? And we shouldn't substitute it with `pkg_config` variable, which can be, for example, something like `/usr/local/bin/pkg-config`?